### PR TITLE
add link to docs in readme

### PR DESCRIPTION
**Pull request by shanamatthews**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,17 @@ Wallet SDK is open-source and uses minimal dependencies for maximum security
 and no code bloat. Simply drop a few lines of code into your dapp and Wallet 
 SDK takes care of the rest. 
 
+## Getting started
+
+- This readme cointains brief instructions to get up and running.
+- Visit our [public developer docs](https://docs.cloud.coinbase.com/wallet-sdk/docs) for more detail, including samples for integating Coinbase Wallet using libraries like [web3-react](https://github.com/NoahZinsmeister/web3-react) and [web3modal](https://github.com/Web3Modal/web3modal).
```
Comment (ewerx) [inline README.md:19]: can we replace web3modal with wagmi or include them all?

**Review by ewerx — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for ewerx @ 2022-03-10 14:39:33 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Comment by thnwatrthitiwitythnathnath** [comment]
Hi

**Comment by Hlllllllii** [comment]
🇺🇸🪄🇬🇧


Sent from Yahoo Mail for iPhone


On Thursday, March 10, 2022, 3:39 PM, Heimdall ***@***.***> wrote:




Review Error for ewerx @ 2022-03-10 14:39:33 UTC
User failed mfa authentication, public email is not set on your github profile. see go/mfa-help

**Pull request merged**